staging/src/k8s.io/code-generator/cmd/conversion-gen/main.go

```diff
@@ -14,20 +14,38 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// conversion-gen is a tool for auto-generating Conversion functions.
+// conversion-gen is a tool for auto-generating functions that convert
+// between internal and external types.  A general conversion task
+// involves three packages: the one containing the internal types, the
+// one containing the external types, and the destination (i.e., the
+// one containing the conversion functions).
 //
-// Given a list of input directories, it will scan for "peer" packages and
-// generate functions that efficiently convert between same-name types in each
-// package.  For any pair of types that has a
-//     `Convert_<pkg1>_<type>_To_<pkg2>_<Type()`
-// function (and its reciprocal), it will simply call that.  use standard value
-// assignment whenever possible.  The resulting file will be stored in the same
-// directory as the processed source package.
+// For each conversion task, `conversion-gen` will generate functions
+// that efficiently convert between same-name types in each of the two
+// packages.  The generated functions include ones named
+//     autoConvert_<pkg1>_<type>_To_<pkg2>_<type>
+// for each such ordered pair of types.  If the destination package
```

Comment:
Could you clarify what you mean by "ordered pair"?

Reply:
Reworded.